# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/powerpc-utils/makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- pmac-utils.orig/Makefile	2003-04-21 00:47:34.000000000 +0200
+++ pmac-utils/Makefile	2003-04-21 00:55:32.000000000 +0200
@@ -27,7 +27,7 @@
 INSTALL	= /usr/bin/install -c
 SOUND_INC = -I.
 
-PROGS	= clock mousemode nvsetenv trackpad backlight nvsetvol
+PROGS	= clock fdeject mousemode nvvideo nvsetenv trackpad backlight nvsetvol sndvolmix vmode
 SCRIPTS	= macos
 
 GENERATED_MAN = clock.8 macos.8 nvsetenv.8 sndvolmix.8 fdeject.1 \
@@ -124,24 +124,13 @@
 		chmod 600 $(DESTDIR)$(devsdir)/sndstat; \
 	fi
 
-install:	all man installdirs
+install:	all installdirs
 	$(INSTALL) -m 4511  clock $(DESTDIR)$(sbindir) 
 	$(INSTALL) -m 755   nvsetenv nvvideo $(DESTDIR)$(sbindir)
 	$(INSTALL) -m 755   mousemode sndvolmix vmode nvsetvol $(DESTDIR)$(usbindir)
 	$(INSTALL) -m 755   fdeject $(DESTDIR)/$(ubindir)
 	$(INSTALL) -m 755   $(SCRIPTS) $(DESTDIR)$(sbindir)
 
-	$(INSTALL) -m 644   fdeject.man	$(DESTDIR)$(man1dir)/fdeject.1
-	$(INSTALL) -m 644   clock.man	$(DESTDIR)$(man8dir)/clock.8
-	$(INSTALL) -m 644   mousemode.man	$(DESTDIR)$(man8dir)/mousemode.8
-	$(INSTALL) -m 644   nvsetenv.man	$(DESTDIR)$(man8dir)/nvsetenv.8
-	$(INSTALL) -m 644   nvvideo.man	$(DESTDIR)$(man8dir)/nvvideo.8
-	$(INSTALL) -m 644   sndvolmix.man	$(DESTDIR)$(man8dir)/sndvolmix.8
-	$(INSTALL) -m 644   vmode.man	$(DESTDIR)$(man8dir)/vmode.8
-	$(INSTALL) -m 644   macos.man	$(DESTDIR)$(man8dir)/macos.8
-	$(INSTALL) -m 644   trackpad.8	$(DESTDIR)$(man8dir)/trackpad.8
-	$(INSTALL) -m 644   nvsetvol.8	$(DESTDIR)$(man8dir)/nvsetvol.8
-
 cleanobjs:
 	$(RM) *.o *.bak *~ *.man $(GENERATED_MAN)
 
